cajita/src/CMakeLists.txt

```diff
@@ -23,6 +23,10 @@ set(HEADERS_PUBLIC
   Cajita_UniformDimPartitioner.hpp
   )
 
+if(Kokkos_ENABLE_OPENMPTARGET) #FIXME_OPENMPTARGET
+  list(REMOVE_ITEM HEADERS_PUBLIC Cajita_Parallel.hpp)
```

Comment:
Is the issue here the multidimensional parallelism?

Reply:
Yeah. So that affects quite a bit of Cajita, but this will at least enable what CabanaMD needs